Amethyst/Events/HotKeyManager.swift

```diff
@@ -323,9 +324,15 @@ final class HotKeyManager: NSObject {
         hotKeyNameToDefaultsKey.append(["Swap focused window with main window", CommandKey.swapMain.rawValue])
         hotKeyNameToDefaultsKey.append(["Force windows to be reevaluated", CommandKey.reevaluateWindows.rawValue])
 
-        (1..<10).forEach { spaceNumber in
+        (1...10).forEach { spaceNumber in
             let name = "Throw focused window to space \(spaceNumber)"
-            hotKeyNameToDefaultsKey.append([name, "\(CommandKey.throwSpacePrefix.rawValue)-\(spaceNumber)"])
+
+            // Need to map spaceNumber 10 to throw-space-0 to preserve order of hotkeys
```

Comment:
Doing `(0..<10)` would affect the order of hotkeys in the application preferences:
![bildschirmfoto 2017-05-11 um 17 37 50](https://cloud.githubusercontent.com/assets/247852/25977482/ba6594fc-3670-11e7-913a-2504ab953570.png)